charts/core/values.yaml

```diff
@@ -64,11 +64,11 @@ autoGenerateCert: true
 
 defaultValidityPeriod: 365
 
-internal: 
+internal:
   certmanager: # enable when cert-manager is installed for the internal certificates
     enabled: false
-    secretname: neuvector-internal
   autoGenerateCert: true
+  secretname: neuvector-internal
```

Comment:
This value arguable shouldn't be indented since it is set if `certmanager` is enabled OR if `autoGenerateCert` is true